lib/MusicBrainz/Server/Edit/Relationship/AddLinkAttribute.pm

```diff
@@ -9,10 +9,10 @@ use MusicBrainz::Server::Entity::Util::JSON qw( to_json_object );
 use MusicBrainz::Server::Translation qw( N_lp );
 
 extends 'MusicBrainz::Server::Edit';
-with 'MusicBrainz::Server::Edit::Relationship',
-     'MusicBrainz::Server::Edit::Role::Insert',
+with 'MusicBrainz::Server::Edit::Role::Insert',
      'MusicBrainz::Server::Edit::Role::AlwaysAutoEdit';
 
+sub edit_category { l('Relationship') }
```

Comment:
Was this really the only purpose of Edit::Relationship here? :melting_face:

Reply:
Not *quite* the only purpose :) `Undefined subroutine &MusicBrainz::Server::Edit::Relationship::EditLinkType::l`. Will amend.